daprdocs/content/en/reference/components-reference/supported-bindings/blobstorage.md

```diff
@@ -201,6 +202,82 @@ To perform a get blob operation, invoke the Azure Blob Storage binding with a `P
 
 The response body contains the value stored in the blob object.
 
+### Delete Blob
```

Comment:
```suggestion
### Delete blob
```

Reply:
Updated the file according to your suggestion.